### ISSUE TITLE
Proposed rule: prefer-get-or

### ISSUE THREAD

**Issue by gunar** [jfmengels/eslint-plugin-lodash-fp#31]
You could warn the users agains
```js
_.get('prop', obj) || default
```
in favor of
```js
_.getOr('prop', obj, default)
```

**Comment by jfmengels** [comment]
Hmm... It's not exactly the same thing, as I learned recently (see https://github.com/lodash/lodash/issues/2375)

the `||` operator uses the right-side value when the left-side one is falsy, but for `getOr`, only if it is `undefined`

```js
const object = {a: null};
_.get('a', object) || 100;
// => 100
_.getOr('a', object, 100);
// => null

// Same when the found value is 0
```

Let me know what you think.

**Comment by gunar** [comment]
You're right. Warning users of a assign three arguments to `_.get` should be enough. However, I still think `getOr` is kind of hidden in the docs. 

Thanks!

**Comment by jfmengels** [comment]
Yeah. it's not even available in the generated docs (https://github.com/jfmengels/lodash-fp-docs) (which I'll start working on at some point).
That's why I'm pretty happy with the fact that the plugin tells you about it, but it should definitely be in the docs, its not enough.

**Comment by gunar** [comment]
Looking forward to that!
